<launch>

  <node name="guts_sonar_ir_node" pkg="guts" type="guts_sonar_ir_node.py" />
  <node name="guts_sonar_gp_node" pkg="guts" type="guts_sonar_gp_node.py" cwd="node" />
  <node name="guts_sonar_gp_rviz" pkg="guts" type="guts_sonar_gp_rviz.py" />
  <node name="rviz" pkg="rviz" type="rviz" />
</launch>
